src/docfx/build/metadata/UserMetadata.cs

```diff
@@ -55,6 +55,7 @@ internal class UserMetadata
         [JsonProperty("_tocRel")]
         public string? TocRel { get; init; }
 
+        [JsonProperty("robots")]
```

Comment:
revert? We tend to only mark `JsonProperty` for property names that does not follow the convention.